run_nwaku.sh

```diff
@@ -36,12 +36,12 @@ exec /usr/bin/wakunode\
       --rest-private=true\
       --rest-address=0.0.0.0\
       --rln-relay=true\
-      --rln-relay-dynamic=false\
+      --rln-relay-dynamic=true\
```

Comment:
should these parameters also be added in this script:
-rln-relay-eth-client-address 
-rln-relay-eth-contract-address
?

Reply:
yes, otherwise nwaku wont know which contract/blockchain to use. and wont be able to sync the rln memberships.

Reply:
added in commit 36d046b